docs/main/prerequesites.md

```diff
@@ -77,4 +77,17 @@ firewall-cmd --permanent --zone=ftl --add-interface=lo
 firewall-cmd --permanent --zone=ftl --add-port=4711/tcp
 firewall-cmd --reload
 ```
+
+#### ufw
+
+ufw stores all rules persistent, so you just need to execute the commands below. If only IPv4 blocking is used for the Pi-hole installation, the rule `546:547/udp` can be removed from the commands below.
+
+```bash
+ufw allow 80/tcp
+ufw allow 53/tcp
+ufw allow 53/udp
+ufw allow 67/tcp
+ufw allow 67/udp
+ufw allow 546:547/udp
+```
```

Comment:
Thanks, can you break this in to two sections to be more explicit with IPv4 vs IPv6. My concern is that people will copy.paste the whole block and not read the caveat in the text. Something along the lines of:

#### ufw

ufw stores all rules persistent, so you just need to execute the commands below. 

IPv4:

```bash
ufw allow 80/tcp
ufw allow 53/tcp
ufw allow 53/udp
ufw allow 67/tcp
ufw allow 67/udp
```

IPv6 (include above IPv4 rules):

```bash
ufw allow 546:547/udp
```

Reply:
Thanks for your request, I changed my pull request according to your suggestion.